include/host-build.mk

```diff
@@ -206,7 +206,7 @@ ifndef DUMP
 
     ifneq ($(CONFIG_AUTOREMOVE),)
       host-compile:
-		$(FIND) $(HOST_BUILD_DIR) -mindepth 1 -maxdepth 1 -not '(' -type f -and -name '.*' -and -size 0 ')' | \
+		$$(call find_depth,$(HOST_BUILD_DIR),'!' '(' -type f -name '.*' -size 0 ')',1,1) | \
```

Comment:
`-not` might not be POSIX, but I believe support is widespread.  I strong dislike use of `'` for single-characters, I strongly prefer `\!`, `\(`, and `\)` instead.  I will admit opinions may differ though.